app/commands/solution/publish.rb

```diff
@@ -2,10 +2,12 @@ class Solution
   class Publish
     include Mandate
 
-    initialize_with :solution, :iteration_idx
+    initialize_with :solution, :user_track, :iteration_idx
```

Comment:
I pass in the user_track instead of doing `solution.user_track` to be consistent with `Solution::Complete`

Reply:
Is that preferable to doing solution.user_track inside `Solution::Complete` do you think?

Reply:
Well, we already have the `user_track` instantiated before `Solution::Complete` is called to do another check. I was slightly hesistant to use `solution.user_track` due to this comment: https://github.com/exercism/website/blob/main/app/models/solution.rb#L12

Reply:
Yeah, pass it in.